### ISSUE TITLE
JS reloading of page with `ajaxLoad` set to False fails accessibility - WCAG SC 3.2.2 (On Input)

### ISSUE THREAD

**Issue by JeffersonBledsoe** [collective/collective.collectionfilter#121]
The `ajaxLoad` setting, when set to false, redirects the page in order to show the new content when a collection filter or search is changed. This fails the [WCAG A Success Criterion 3.2.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/consistent-behavior-unpredictable-change.html) as a context changed is caused by the onchange event. I do not believe this applies when `ajaxLoad` is set to true, as this changes the contents of the page rather than the context.
